<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameGZ</class>
 <widget class="QFrame" name="FrameGZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxShowGZ">
     <property name="maximumSize">
      <size>
       <width>208</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Show Guard zone</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxAlarmGZ">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>208</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Enable Notif</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonArc">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Arc</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonCircle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Circle</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Inner range :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditInner">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>60.1</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Outer range :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOuter">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Start bearing :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditStart">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>deg</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>End bearing :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditEnd">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>360.23</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>deg</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Notif threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNotifTHR">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>10-500</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
